package.json

```diff
@@ -7,7 +7,7 @@
     "client": "vite --config vite.config.js",
     "clientinstall": "npm install --prefix client",
     "server": "node index.js",
-    "start": "concurrently 'npm run server' 'npm run client'",
+    "start": "concurrently \"npm run server\" \"npm run client\"",
```

Comment:
You can remove this, I have already raised PR for this here https://github.com/finos/git-proxy/pull/784